doc/docs/cli-fetch.md

```diff
@@ -59,6 +59,23 @@ $ cs fetch --javadoc com.lihaoyi:ammonite_2.12.8:1.6.0
 …
 ```
 
+## File arguments
+
+Pass `--dependency-file <filename>` to read dependecies from file. 
+File content should a list of dependencies separated by a newline character.
+Example usage:
+```bash
+$ cs fetch --dependency-file file_1 --dependency-file file_2
+/Users/jciesluk/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/circe/circe-generic_2.13/0.12.3/circe-generic_2.13-0.12.3.jar
+/Users/jciesluk/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/lihaoyi/ammonite_2.12.8/1.6.0/ammonite_2.12.8-1.6.0.jar
+…
+```
+With `file_1` content:
+```
+io.circe::circe-generic:0.12.3
+com.lihaoyi:ammonite_2.12.8:1.6.0
+```
```

Comment:
In this example, the fact that there is no `file_2` is not an error?

Reply:
I wanted to use two files in command to show that user can do that, but I gave example content of just one file. The second file must also exist. I've added example content of `file_2` so it should be clear now